src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+
+import './Carousel.css'
+
+class Carousel extends Component {
+    state = {
+        position: 0
+    }
+
+    prev() {
+        if ((this.state.position - 1) < 0){
+            this.setState({
+            position: ((this.props.images.length) - 1)
```

Comment:
si haces destructuring de images al inicio del componente ya no tendrás que usarla como `this.props.images` en cada sitio